src/main/java/edu/kis/powp/jobs2d/drivers/visitor/DriversCountVisitor.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.powp.jobs2d.drivers.visitor;
+
+import edu.kis.powp.jobs2d.drivers.VisitableJob2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.DriverDecorator;
+import edu.kis.powp.jobs2d.drivers.composite.IDriverComposite;
+import edu.kis.powp.jobs2d.drivers.logger.LoggerDriver;
+
+import java.util.List;
+
+public class DriversCountVisitor implements IDriverVisitor {
+    private long count = 0;
+
+    public long getCount() {
+        return count;
+    }
+
+    @Override
+    public void visit(IDriverComposite driver) {
+        count = 0; // Reset
+        List<VisitableJob2dDriver> drivers = driver.getDrivers();
+        drivers.forEach(d -> d.accept(this));
+        count += 1;
```

Comment:
the loop should perform cummulation
for(VisitableJob2dDriver d: driver.getDrivers()) {
    d.accept(this);
    tmpCounter += count; 
}

tmpCounter += count is how we can simulate return value. accept stores its result in count field, we can use it right away. Finally, we set the result, e.g.: count = tmpCounter + 1.